### DISCUSSION TITLE
Create an spanish version.

### DISCUSSION THREAD

**Discussion by claretdevigne** [TheOdinProject/curriculum#25126] [Ideas]
I would like helping people and collaborate with The Odin Project by creating an Spanish version of the website. This can be useful for a lot of people. I'd like helping with this but not without asking first.

**Comment by CouchofTomato** [comment]
Hey @claretdevigne 

Thanks for reaching out with your idea. Although we'd love to support additional languages, there are two pretty big barriers.

1) We use a lot of external resources which might not have a decent counterpart in another language.
2) We have to maintain it. Everytime we make a change to the English version this would ripple down to any other language we support. We just don't have the manpower.

If there is ever a way we can make it happen and be supported in the long term we would. We want to make our course as accessible as possible. Hopefully that reality isn't far away.
